.github/workflows/test-microk8s.yaml

```diff
@@ -0,0 +1,83 @@
+name: Charmed Katib
```

Comment:
Should you name this file `test-microk8s.yml` ?
I don't see that this action is running on this PR.

Reply:
Updated the name of the file to match this name. As far as why it's not showing up, do you have to enable actions in settings or anything? I'm not sure exactly why it's not showing up.

Reply:
@Bobgy Can you help us with enabling GitHub actions on the Katib repository, please ?
I think it should be pretty straightforward: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/disabling-or-limiting-github-actions-for-a-repository#managing-github-actions-permissions-for-your-repository
/cc @gaocegege @johnugeorge